### PR TITLE
workloadccl: add a `fixtures import` flag for experimental faster IMPORT

### PR DESCRIPTION
**Pull request by danhhz**
Release note: None

### CODE REVIEW

**Comment by danhhz** [comment]
Depends on the PR that adds the experimental faster import https://github.com/cockroachdb/cockroach/pull/34751

File: pkg/ccl/workloadccl/fixture.go [hunk 1]
```diff
@@ -468,6 +470,9 @@ func importFixtureTable(
 		params = append(params, output)
 		fmt.Fprintf(&buf, `, transform=$%d`, len(params))
 	}
+	if directIngestion {
+		buf.WriteString(`, direct_ingestion`)
```
Comment (dt) [inline pkg/ccl/workloadccl/fixture.go]: this got an experimental prefix by the time it merged

**Review by dt — APPROVED** [review]
#34751 merged

**Review by danhhz — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/34790)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @dt)

___
*[pkg/ccl/workloadccl/fixture.go, line 474 at r1](https://reviewable.io/reviews/cockroachdb/cockroach/34790#-LYT57MSnAAi2CVtjJn7-r1-474:-LYharNFA9uOkTONV0wW:b-896fix) ([raw file](https://github.com/cockroachdb/cockroach/blob/6ccd383a1deb4ae535117c591f0d8c21694b0e98/pkg/ccl/workloadccl/fixture.go#L474)):*
<details><summary><i>Previously, dt (David Taylor) wrote…</i></summary><blockquote>

this got an experimental prefix by the time it merged
</blockquote></details>

Done.

**Comment by danhhz** [comment]
bors r=dt

**Pull request merged**